packages/cli/src/cmds/validator/signers/index.ts

```diff
@@ -86,7 +106,20 @@ export async function getSignersFromArgs(args: IValidatorCliArgs & IGlobalArgs,
 
     // Read and decrypt local keystores, imported via keymanager api or import cmd
     const keystoreDefinitions = persistedKeysBackend.readAllKeystores();
-    const keystoreSigners = await decryptKeystoreDefinitions(keystoreDefinitions, args);
+
+    const needle = showProgress({
+      total: keystoreDefinitions.length,
+      frequencyMs: KEYSTORE_IMPORT_PROGRESS_MS,
+      signal: signal,
+      progress: ({ratePerSec, percentage, current, total}) => {
+        logger.info(
+          `${percentage.toFixed(0)}% of local keystores imported. current=${current} total=${total}${current} rate=${(
+            ratePerSec * 60
+          ).toFixed(2)}keys/m`
```

Comment:
This duplicates the same code from lines 82-93 btw. Might be worth putting in a function.